requirements.txt

```diff
@@ -11,4 +11,3 @@ gunicorn
 django_compressor==2.0
 celery<4.0
 django-google-analytics-app==2.1.6
-Unidecode==0.04.16
```

Comment:
@codiebeulaine I had reverted to this running lower version and re-testing.  
Adding all reverts back.